Add return type annotation for ShardedTensor (#19)

colin2328 opened the pull request:
Summary: Pull Request resolved: https://github.com/fairinternal/torchrec/pull/19

Test Plan: CI

Reviewed By: wanchaol

Differential Revision: D32502393



cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @SciPioneer @H-Huang

pritamdamania87 reviewed (approved):
Thanks for fixing this!

pritamdamania87 commented:
Please look through the lint failure though!

colin2328 committed:
Add return type annotation for ShardedTensor (#69945)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/69945

Test Plan: CI

Reviewed By: wanchaol

Differential Revision: D32502393

fbshipit-source-id: 8b2fa0da1ba38ba86043ef9c65b4146e3b147f14
